pkg/utils/build/hacbs.go

```diff
@@ -0,0 +1,65 @@
+package build
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func CheckSkipResult(taskRefName string, skiplist []string) bool {
+	for _, b := range skiplist {
+		if taskRefName == b {
+			return true
+		}
+	}
+	return false
+}
+
+func FetchTaskRunResult(pr *v1beta1.PipelineRun, pipelineTaskName string, result string) (string, error) {
+	for _, tr := range pr.Status.TaskRuns {
+		if tr.PipelineTaskName != pipelineTaskName {
+			continue
+		}
+		for _, trResult := range tr.Status.TaskRunResults {
+			if trResult.Name == result {
+				return strings.TrimSuffix(trResult.Value, "\n"), nil
+			}
+			if trResult.Name == "BASE_IMAGE_REPOSITORY" || trResult.Name == "BASE_IMAGE" {
```

Comment:
I think that hardcoding task result name that's different from what's requested in the `result` parameter could cause confusion when making future changes. 
Calling this function with different parameters (e.g. with 'BASE_IMAGE') when needed would make this simpler to follow.

Reply:
done

Reply:
Thanks!